### ISSUE TITLE
Cannot leave one-on-one chat room

### ISSUE THREAD

**Issue by ThatGeoGuy** [gitterHQ/gitter#1572]
Seems there is no "leave" option for one-on-one chats. Can this be fixed? Hiding only hides the chat visually, but from the API I still seem to see the chat existing (I would prefer it not be there, crowding selections). 

The steps that I took to get this were: 

1. Searched for vim
2. Clicked on a user by mistake
3. Now I have a chat with this user open in API that won't go away. 

I didn't intend to start a one-on-one with a user, but now their username is in my list of open rooms. Can they be removed?

**Comment by ThatGeoGuy** [comment]
Yes, I've noticed that I can only hide 1-to-1 rooms. That is indeed the problem that I reported. The problem is that those rooms show up even if you hide them when you search with the API.

**Comment by MadLittleMods** [comment]
They should appear if you are using the search endpoints. If you are referring to the `/v1/rooms` endpoint then you can filter by whether `lastAccessTime` exists on the room object.
